### PR TITLE
improve(ProfitClient): Safe default for minRelayerFeePct

### PR DESCRIPTION
**Pull request by pxrl**
Third-party relayers should not need to configure this to avoid filling
unprofitable relays.

### CODE REVIEW

File: src/common/Constants.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@
 // in the HubPool's proposeRootBundle method should be: Mainnet, Optimism, Polygon, Boba, Arbitrum
 export const CHAIN_ID_LIST_INDICES = [1, 10, 137, 288, 42161];
 
+export const RELAYER_MIN_FEE_PCT = 0.0003;
```
Comment (pxrl) [inline src/common/Constants.ts:5]: Have retained the existing name (*_PCT) for this new constant, but we're not actually specifying a percentage here. Should we change it so the user configures an actual percentage (i.e. .03) and then handle the division by 100 internally?

Comment (mrice32) [reply src/common/Constants.ts:5]: Personally, I don't really see anything wrong with labeling something a percentage even if 100% is represented by 1. However, maybe the more accurate phrasing would be "fraction".

Either way, this LGTM

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/common/Constants.ts (hunk 1 above)
Comment (pxrl) [reply src/common/Constants.ts:5]: It's not contentious, so I'll leave it as-is. Might revisit later if I develop strong feelings either way.

**Pull request merged**